jenkins/test_dendron.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -u
+
+: ${GOPATH:=${WORKSPACE}/.gopath}
+if [[ "${GOPATH}" != *:* ]]; then
+  mkdir -p "${GOPATH}"
+  export PATH="${GOPATH}/bin:${PATH}"
+fi
+export GOPATH
+
+go get github.com/constabulary/gb/...
```

Comment:
This is duplicated so that build_dendron and test_dendron are independent of each other.